### PR TITLE
Update Node.md with short intro.

### PR DESCRIPTION
**Pull request by creatinglake**

### CODE REVIEW

File: docs/ecosystem/nodes.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
-# Topic
 
-TBD
+# Veres Ones Nodes
+Nodes are responsible for providing computational and storage
```
Comment (msporny) [inline docs/ecosystem/nodes.md]: Add a space between the title and the content to ease readability for devs.

**Review by msporny — CHANGES_REQUESTED** [review]

**Comment by mattcollier** [comment]
@creatinglake welcome to github!

**Comment by creatinglake** [comment]
@mattcollier  thanks!

**Comment by msporny** [comment]
LGTM

**Pull request merged**